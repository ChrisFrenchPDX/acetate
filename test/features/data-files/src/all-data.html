---
data:
  dynamic: dynamic-data.js
  json: json-data.json
  yaml: yaml-data.yaml
  yml: yml-data.yml
---

{{ data.dynamic.value }} {{ data.json.value }} {{ data.yaml.value }} {{ data.yml.value }} {{ data.global.value }}