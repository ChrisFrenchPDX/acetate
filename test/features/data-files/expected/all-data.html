dynamic json yaml yml global